cmd/drand-gossip-relay/client/client.go

```diff
@@ -0,0 +1,124 @@
+package client
+
+import (
+	"context"
+	"sync"
+
+	"github.com/drand/drand/cmd/drand-gossip-relay/lp2p"
+	"github.com/drand/drand/protobuf/drand"
+	"github.com/gogo/protobuf/proto"
+	logging "github.com/ipfs/go-log/v2"
```

Comment:
consider switching to `github.com/drand/drand/log` since that seems to be where the rest of the repo is converging to

Reply:
libp2p will log to `github.com/ipfs/go-log/v2`, and this doesn't use much of any drand comonents.

Reply:
I can switch it in future.